release-notes.txt

```diff
@@ -1,3 +1,46 @@
+OpenMDAO Version 1.6.0 Alpha Release Notes
+March 1, 2016
+
+Backwards-Incompatible Change
+*System names must be valid Python names:
+ Because we add subsystems as attributes of their parent system, the name of each system must be a valid Python name. Otherwise we will raise an exception.
+
+Features
+*User now has the ability to call check_total_derivs and have it just do the driver-relevant params/cons/objs instead of every param/unknown in the model.
+*Multiple local components are now allowable in a parallel system.
+*User can complex step for total derivatives.
+*Created an NPSS wrapper for OpenMDAO 1.0.
```

Comment:
Should we mention the NPSS wrapper in the OpenMDAO release notes since it's in a separate repo?